### PR TITLE
Ticket fixes

### LINKED ISSUE #206: [BUG] Ticket Activity Listener Problem
**Describe the bug**
The bot still asks to delete when there are users on the voice channels. This should not be the case. The activity listeners should also check to make sure the room is still alive with the channels.

**To Reproduce**
Steps to reproduce the behavior on Discord:
1. With a cave or advanced ticket system, open a ticket
2. join the voice channel
2. wait for the ask delete console (which should happen)
3. Then remove the channels manually and errors will continue to appear

**Expected behavior**
Errors should not come up (unknown channel) and bot should not ask to delete when users are in the voice channels.

**Discord Platform**
Discord App and Web App

### PR DESCRIPTION
**Pull request by mwang2000**
# Description
Bug Fix - closes #206 
Previously, the code was written so that after the first time the bot asks to delete a ticket, and a user tells it that they still want to keep it, it will keep asking again at the next interval without checking if people have been using it. I fixed this by routing it back through startChannelActivityListener() which does essentially the same thing but it also checks for whether people have been using the text and voice channels.

Also while testing the ticket system I found some small bugs where Features were not being added to embeds/consoles, which were caused by passing a regular Object into addFeature() instead of converting them into Features first. Another bug occurred when someone submitted their ticket request, it would cause an error because strings have no Mentions property so I changed StringPrompt to MessagePrompt.

Next I would def be checking the rest of the Features to see that they are created as Features and not just passed in as Objects.

## Why
🐛 

## User Changes
none

## Tests Done
manual testing

# Breaking Changes

### CODE REVIEW

File: classes/Bot/Features/Ticket_System/ticket.js [hunk 1]
```diff
@@ -273,31 +272,33 @@ class Ticket {
...
+            Feature.create({
+                name: 'When done:',
+                description: `React to this message with ${leaveTicketEmoji} to lose access to these channels!`,
+                emoji: leaveTicketEmoji,
+                callback: (user, reaction, stopInteracting) => {
+                    // delete the mentor or the group member that is leaving the ticket
+                    this.helpers.delete(user.id);
+                    this.group.delete(user.id);
+
+                    this.room.removeUserAccess(user);
```
Comment (christylo) [inline classes/Bot/Features/Ticket_System/ticket.js:285]: maybe consider refactoring this (ie. extracting into another function) to follow SRP?

**Review by christylo — APPROVED** [review]
lgtm just a single comment :))

**Review by MrBenC88 — APPROVED** [review]
nice job!

**Review by JPGarCar — APPROVED** [review]
Awesome will merge!

**Pull request merged**